server/app/services/rpc/node_handler.rb

```diff
@@ -7,6 +7,11 @@ class NodeHandler
 
     def initialize(grid)
       @grid = grid
+      @db_session = ContainerLog.collection.session.with(
```

Comment:
The `ContainerLog` looks a bit odd here, but I suppose it doesn't really make any difference. Maybe use `HostNode` instead, though?